### PR TITLE
Provide Scope Edit Capability

### PR DESCRIPTION
**Pull request by jamesduncan**

### CODE REVIEW

**Review by nh758 — APPROVED** [review]

File: patches/20230324.sql [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# ************************************************************
+# 20230324.sql
+# 
+# This patch allows Scopes to be edited in the Site Admin App
+# It also makes some minor changes to the UI to update look 
+# and make use of new widgets for filtering.
+# ************************************************************
+
+#
+# 1) Insert the Switcheroo Role
```
Comment (Hiro-Nakamura) [inline patches/20230324.sql:10]: Is this what you are doing here?

File: patches/20230324.sql [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+	('ffff06b8-fdf0-4dfb-bc42-dd20d52ceb53','Roles.layout.grid','view','{\"id\":\"ffff06b8-fdf0-4dfb-bc42-dd20d52ceb53\",\"type\":\"view\",\"key\":\"grid\",\"icon\":\"table\",\"name\":\"Roles.layout.grid\",\"settings\":{\"columnSpan\":1,\"rowSpan\":1,\"dataviewID\":\"bbef27bb-4673-468a-8c7b-ddd8c6454f19\",\"isEditable\":0,\"massUpdate\":0,\"allowDelete\":1,\"isSortable\":0,\"isExportable\":0,\"hideHeader\":0,\"labelAsField\":0,\"hideButtons\":1,\"detailsPage\":\"\",\"editPage\":\"abae9cec-6ec0-4191-9f72-8f85f6197e38\",\"detailsTab\":\"\",\"editTab\":\"\",\"objectWorkspace\":{\"frozenColumnID\":\"\",\"hiddenFields\":[\"scopes\",\"ProcessForm30\"]},\"height\":0,\"groupBy\":\"\",\"gridFilter\":{\"filterOption\":0,\"queryRules\":[],\"isGlobalToolbar\":0},\"padding\":17,\"showToolbar\":1,\"columnConfig\":[],\"saveLocal\":1,\"trackView\":0,\"frozenColumnID\":\"\",\"hiddenFields\":[\"scopes\",\"ProcessForm30\"],\"summaryColumns\":[],\"countColumns\":[]},\"accessLevels\":{},\"translations\":[{\"language_code\":\"en\",\"label\":\"Roles.layout.grid\"}],\"viewIDs\":[],\"position\":{\"dx\":1,\"dy\":1},\"isRoot\":false}','2020-12-01 09:23:38','2023-03-10 10:00:17')
+ON DUPLICATE KEY UPDATE
```
Comment (Hiro-Nakamura) [inline patches/20230324.sql:90]: Nice SQL here ...

**Review by Hiro-Nakamura — CHANGES_REQUESTED** [review]
Just update the comment and you're good to go.

File: patches/20230324.sql (hunk 1 above)
Comment (jamesduncan) [reply patches/20230324.sql:10]: Haha! Thanks...changing it now.

**Pull request merged**